### DISCUSSION TITLE
Image size limitation

### DISCUSSION THREAD

**Discussion by Tembhum** [goharbor/harbor#16650] [Q&A] [answered]
Are there any image size limitation on Harbor? Can it handle 10-15 GB container image and are there any size limitation on each layer level?

**Comment by wy65701436** [comment]
So far no.

**Reply by Vad1mo** [reply]
@wy65701436 there is actually https://github.com/goharbor/harbor/pull/16322

**Comment by Vad1mo** [comment, marked as answer]
Currently Harbor does not support layer above 10GB.
Sometime windows images or datasets have layers that size. 

See this open PR: https://github.com/goharbor/harbor/pull/16322
